recipes/odbc/all/conanfile.py

```diff
@@ -57,7 +57,8 @@ def _configure_autotools(self):
         args = ["--enable-static=%s" % static_flag,
                 "--enable-shared=%s" % shared_flag,
                 "--enable-ltdl-install",
-                "--enable-iconv=%s" % libiconv_flag]
+                "--enable-iconv=%s" % libiconv_flag,
+                "--sysconfdir=/etc"]
```

Comment:
Is it always `/etc/` on all Linux-like or BSD-like machines?

Reply:
That is the default

Reply:
When you install it as a system package it includes shared libs and the config file location is `/etc`.  This is making the Conan-provided static libs compatible.  The other alternative is to create an option for this and make the default `/etc`.

Reply:
I think having `/etc` is okay.
If someone really wants to change the value, they will create PR allowing any path.

Reply:
I would use self.package_folder/res, but it's complicated to move configuration files from a system to Conan, so /etc is fine for me.